docs/pages/machine-id/troubleshooting.mdx

```diff
@@ -292,3 +292,46 @@ flag:
 $ tctl bots rm example
 $ tctl bots add example --roles=foo,bar,machine-id-db
 ```
+
+## Kubernetes connections are failing with `Unable to connect to the server: x509: certificate signed by unknown authority`
+
+### Symptoms
+
+A self-hosted cluster is connecting Machine ID to Kubernetes clusters with the following errors.  This can happen for non-TLS configured 
```

Comment:
Since 'cluster' is overloaded, should we be specific?

```suggestion
A self-hosted Teleport cluster is connecting Machine ID to Kubernetes clusters with the following errors.  This can happen for non-TLS configured 
```